src/message_filter.rs

```diff
@@ -6,38 +6,16 @@
 // KIND, either express or implied. Please review the Licences for the specific language governing
 // permissions and limitations relating to use of the SAFE Network Software.
 
-use crate::messages::{Message, MessageHash};
 use lru_time_cache::LruCache;
-use sn_messaging::DstLocation;
+use sn_messaging::MessageId;
 use std::time::Duration;
-use xor_name::XorName;
 
 const INCOMING_EXPIRY_DURATION: Duration = Duration::from_secs(20 * 60);
-const OUTGOING_EXPIRY_DURATION: Duration = Duration::from_secs(10 * 60);
 const MAX_ENTRIES: usize = 5_000;
 
-/// An enum representing a result of message filtering
-#[derive(Eq, PartialEq)]
-pub enum FilteringResult {
-    /// We don't have the message in the filter yet
-    NewMessage,
-    /// We have the message in the filter
-    KnownMessage,
-}
-
-impl FilteringResult {
-    pub fn is_new(&self) -> bool {
-        match self {
-            Self::NewMessage => true,
-            Self::KnownMessage => false,
-        }
-    }
-}
-
 // Structure to filter (throttle) incoming and outgoing messages.
 pub(crate) struct MessageFilter {
-    incoming: LruCache<MessageHash, ()>,
-    outgoing: LruCache<(MessageHash, XorName), ()>,
```

Comment:
What happened to filtering outgoing?

Reply:
as each WireMsg will be given a random msg_id, there is no point to filtering outgoing (based on random msg_id).

Reply:
Why did we filter outgoing previously and if there was a valid reason are we now creating an issue? Or was it always useless?